exercises/prime-factors/prime_factors_test.go

```diff
@@ -26,3 +27,9 @@ func BenchmarkPrimeFactors(b *testing.B) {
 		}
 	}
 }
+
+func ascending(list []int64) func(int, int) bool {
+	return func(ii, jj int) bool {
+		return list[ii] < list[jj]
```

Comment:
why not `i` and `j` ?

Reply:
It's a habit of mine. Easier to identify / search for `ii` than `i`, which appears in many English words.

Reply:
I think it's better, let's change it then good to go. Thanks!